package.json

```diff
@@ -19,6 +19,10 @@
     "type": "git",
     "url": "http://github.com/julien-c/epub.git"
   },
+  "files": [
+    "epub.js",
```

Comment:
these are the only two files actually needed at runtime for the user

Reply:
good idea